docs/install/requirements.md

```diff
@@ -18,9 +18,10 @@ For running and building the application locally you will also need:
 - [Node.js](https://nodejs.org) 14.0.0+ (needed for local installation). We recommend that you install node through [nvm](https://github.com/nvm-sh/nvm).
 - [npm](https://www.npmjs.com/get-npm) < 7.
 - [Cairo](https://invenio-formatter.readthedocs.io/en/latest/installation.html) needed for badges to be properly displayed.
+- [DejaVu Fonts](https://dejavu-fonts.github.io/Download.html) needed for MacOS systems for badges rendering.
```

Comment:
doubt: Pointing to packages seems complex since this is very platform dependent...

In linux there is [dejavu-sans-fonts](https://github.com/inveniosoftware/docker-invenio/blob/master/python3.8/Dockerfile#L36). This is `yum`, can you check in Ubuntu? In OSX, brew has `font-dejavu`, but does not seem to include the sans one... WDYT? simpler and more user friendly to not give so many options and just point to `Download.html`?

Reply:
A couple of months when I installed a new Mac I didn't find any good alternative that just download it. Originally there was homebrew package, but that doesnt' work anymore. Then homebrew has some complicated font management, so in the end this was the easiest for me. I'd be happy if there was a simpler way to install it though.

Reply:
i didnt add the ubuntu option because for ubuntu the doi badges appear right? so i thought it wasne necessary

Reply:
Agreee, let's just leave the pointer to dejavu and document further along if we find better options.